### DISCUSSION TITLE
Want to select all the text when go on the react-select through tab button

### DISCUSSION THREAD

**Discussion by MuditMT** [JedWatson/react-select#5674] [Q&A]
Hey!
I am using the react-select in my project, so I need whenever I go on the react-select through the tab button the selected value(means the text showing in the input) should be automatically highlighted itself or we can say select all the text.
For eg:-
![image](https://github.com/JedWatson/react-select/assets/84302912/df0b38f1-b2a9-4e90-b9fc-413282fbe91d)

**Comment by bidy** [comment]
Same here!
